### ISSUE TITLE
if Lreg < 0?

### ISSUE THREAD

**Issue by TongkunGuan** [BIT-DA/SePiCo#23]
To compute proto_reg:
    contrast_norm = num_classes * np.log(num_classes)
    proto_sim = mean_feat.mm(mean.permute(1, 0).contiguous()) / contrast_temp
    loss = torch.sum(torch.softmax(proto_sim, dim=1).log()) / contrast_norm

reg_weight * proto_reg(feat, mean, contrast_temp, contrast_norm=contrast_norm)

Why loss need < 0? 
I don't understand it, Thanks!
